Automattic/woocommerce-payments
# Load tokenization script on add-payment-method page

#### Changes proposed in this Pull Request

This adds a check for `is_add_payment_method_page()` when deciding whether to load tokenization script.

This is to resolve a bug where we couldn't add a payment method via my-account because this script isn't enqueued but is a dependency of our checkout js.

See p1633419063059700-slack-C026J0AL8NS for more details

#### Testing instructions

- On `feature/stripe_billing`, go to `My Account > Payment methods > Add payment method`. You should notice an inactive input under `Use a new payment method`.
- On this branch, you will not see this issue.

-------------------


**Post merge**
